makefile

```diff
@@ -3,14 +3,17 @@ EASK ?= eask
 
 TESTHOME=/tmp/emacs
 
-ci: build test
+ci: build compile test
 
 package:
 	$(EASK) package
 
 build: package
 	$(EASK) install
 
+compile:
+	$(EASK) compile
```

Comment:
Wouldn't it be more natural to add this to the build stage instead?

Reply:
You mean,

```
build: package
	$(EASK) install
	$(EASK) compile
```

instead of the separate make command?

Reply:
Yea. I mean “compile” sounds like a pretty central part of building something 🙂

Reply:
Done! 👍